packages/terra-form-textarea/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-form-textarea",
+  "main": "lib/Textarea.js",
+  "version": "0.0.0",
+  "description": "Textarea component for building forms",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-form-textarea",
+    "FormTextarea",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.4.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^2.0.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
```

Comment:
You'll have to update these based on @emilyrohrbough's generator changes https://github.com/cerner/generator-terra-module/pull/71

Reply:
Yes we need these updated based on the generator changes so this package's npm scripts will be compatible on Windows.

Reply:
Fixed here. https://github.com/cerner/terra-core/pull/982/commits/cd1b3406beecc6afa02bd2f687034c7cbf190cc5